DesignFile.md

```diff
@@ -0,0 +1,79 @@
+```python
+from aide_design.play import*
+from aide_design.physchem import*
+import math
+Q_Plant=20*u.L/u.s
+Temp_Plant=20*u.degC
+RatioVCOrifice=.62
+Nu=viscosity_kinematic(Temp_Plant)
+print(Nu)
+Max_Num_Holes=10
+Q_Per_Hole=Q_Plant/Max_Num_Holes
+print(Q_Per_Hole)
+#Rows_Per_T_Active_Area= 1 or 2
+#Active_Height_Of_T=.13*u.m
+#Min/Max_Orifices_Per_Row=
+Avail_Head_High_Est=2*u.m
+Avail_Head_Low_Est=1*u.m
+
+FS_Diam=8*u.inch
+Area_Outlet=((FS_Diam/2)**2)*u.pi/4
+#SS_Diam=4*u.inch     SS - Small scale
+FS_FV_Diam=6*u.inch
+#SS_FV_Diam=3*u.inch     SS- Small Scale
+SliderPipeLength=.762*u.m
+Slider_Hole_Diam=1*u.inch
+Area_Slider_Hole=(((Slider_Hole_Diam/2)**2)*u.pi/4)
+print(Area_Slider_Hole.to(u.m**2))
+V_Outlet=Q_Plant/((FS_Diam/2)**2)*u.pi/4
+print(V_Outlet.to(u.m/u.s))
+
+V_Hole=(Q_Per_Hole/Area_Slider_Hole)
+print(V_Hole.to(u.m/u.s))
+Reynolds=re_pipe(Q_Per_Hole, Slider_Hole_Diam, Nu)
+print(Reynolds)
+Range_Slider=1.2*u.m
+Z_Top_Hole=1.343*u.m
+Z_Bot_Hole=.368*u.m
+Range_Slider=1.2*u.m ## needs to be updated
+Z_Top_Hole=1.343*u.m ## and here
+Z_Bot_Hole=.368*u.m ## same here
+
+def K_Minor_Calc(D1,D2):
```

Comment:
What is this? A reduction k value calculation? You need to give your source for this calc. Please use [this module](https://github.com/AguaClara/aide_design/blob/master/aide_design/k_value_of_reductions_utility.py) for the equations (hint `from aide_design import k_value_reductions_utility`)